lib/common/nvpair.c

```diff
@@ -1004,7 +981,11 @@ pcmk_format_nvpair(const char *name, const char *value,
 char *
 pcmk_format_named_time(const char *name, time_t epoch_time)
```

Comment:
Do you feel like it's worth deprecating this, or just leave it until the time lib overhaul?

Reply:
It's already deprecated :)

Reply:
whoops, didn't see the section header :)